style.css

```diff
@@ -0,0 +1,267 @@
+}
+
+.search-bar{
+    border: none;
+    background-color: var(--white);
+}
+
+
+.container-center{
+    min-height: calc(100vh - 362px);
+    padding: 1rem;
+    margin: auto;
+    max-width: fit-content;
+    margin-bottom: 4rem;
+    font-family: 'Poppins', sans-serif;
+}
+
+.container-center h1{
+    margin-left: 1rem;
+    font-weight: 600;
+}
+
+.container-center p{
+    margin-left: 1rem;
+}
+
+.curated-section{
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.card{
+    font-family: 'Poppins', sans-serif;
+    max-height: 22rem;
+    max-width: 16rem;
+    position: relative;
+    display: flex;
+    align-items: flex-end;
+    margin: 1rem;
+    border-radius: 0.5rem;
+    transition: transform 0.4s;
+}
+
+.card:hover{
+    transform: scale(0.97);
+}
+
+.card-img{
+    width: 100%;
+    height: 100%;
+    border-radius: 0.5rem;
+    position: relative;
+}
+
+.card-overlay-text{
+    border-radius: 0.5rem;
+    color: var(--white);
+    width: 100%;
+    position: absolute;
+    padding-left: 1rem;
+    padding-bottom: 0.8rem;
+    background-image: linear-gradient(to top, black, transparent);
+}
+
+.card-overlay-text h4{
+    margin-bottom: 0rem;
+    font-weight: 500;
+}
+
+.card-overlay-text a{
+    color: inherit;
+}
+
+.footer{
+    padding: 1rem 2rem;
+    color: beige;
+    background-color: var(--waterBlue);
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+.footer-about{
+    padding: 0rem 2rem;
+    max-width: 460px;
+    margin: 1rem 2rem;
+}
+
+.footer-about .copyright{
+    margin-top: 2rem;
+    color: var(--gray);
+    font-weight: 500;
+}
+
+.brand-line{
+    font-size: 1.75rem;
+    font-weight: 500;
+    color: var(--veryLightGray);
+}
+
+
+.footer-about span{
+    font-size: 1.9rem;
+    font-weight: 700;
+    letter-spacing: 2px;
+    background-image: linear-gradient(90deg, var(--violet), var(--turquoise));
+    -webkit-background-clip: text;
+    background-clip: text;
+    color: transparent;
+}
+
+.newsletter{
+    margin-top: 1.75rem;
+}
+
+.newsletter p{
+    color: var(--veryLightGray);
+    font-weight: 600;
+}
+
+.newsletter label{
+    margin-bottom: 0.5rem;
+    font-size: 11px;
+    color: var(--gray);
+    font-weight: bold;
+    display: block;
+}
+
+.newsletter-input input{
+    border: none;
+    outline: none;
+    background-color: var(--offWhite);
+    padding: 00.25rem;
+}
+
+.newsletter-input{
+    background-color: var(--offWhite);
+    padding: 0rem 0rem 0rem 0.25rem;
+    border-radius: 0.35rem;
+}
+
+.newsletter-input button{
+    display: inline;
+    padding: 0.5rem;
+    border-radius: 0rem 0.35rem 0.35rem 0rem;
+    background-color: var(--veryLightGray);
+    border: none;
+}
+
+.newsletter-input button:hover{
+    background-color: var(--gray);
+    cursor: pointer;
+}
+
+svg{
+    color: white;
+    width: 20px;
+    height: 20px;
+}
+
+.social-icons{
+    width: 12rem;
+    margin: 1rem;
+    display: flex;
+    justify-content: space-evenly;
+}
+
```

Comment:
I see there's heavy usage of flex properties. A suggestion you can work on if you have the bandwidth right now - create util classes for flex.